src/components/panel/panel.spec.js

```diff
@@ -1194,6 +1239,26 @@ describe('$mdPanel', function() {
       expect(getGroupOpenPanels('test')).not.toContain(panelRef);
     });
 
+    it('should update open panels of all of the panel\'s groups when a panel ' +
+        'is closed', function() {
+          $mdPanel.newPanelGroup('test1');
+          $mdPanel.newPanelGroup('test2');
+
+          var config = {
+            groupName: ['test1', 'test2']
+          };
+
+          openPanel(config);
+          flushPanel();
+          expect(getGroupOpenPanels('test1')).toContain(panelRef);
+          expect(getGroupOpenPanels('test2')).toContain(panelRef);
+
+          closePanel();
+          flushPanel();
```

Comment:
I thought there was a `flushPanel` at the end of the `closePanel`. Why do you need another one?